lucene/directory-provider/src/test/java/org/infinispan/hibernate/search/TwoNodesTest.java

```diff
@@ -60,6 +60,7 @@ public void testSomething() {
       } finally {
          nodeC.close();
       }
+      waitMembersCount(nodea, SimpleEmail.class, 2);
```

Comment:
fix sounds good, but it's a great example of the worst of Infinispan in terms of usability: people starting real nodes on real machines should also invoke such testing helpers?

I shouldn't need to apply such tricks, but to pass some parameter at boot time which expresses how many nodes I'm expecting to have in my cluster: that something that everyone needs.